src/Pages/Overview/OverviewProfile.js

```diff
@@ -17,11 +18,7 @@ export default function OverviewProfile (props) {
   return (
     <tr>
       <td>
-        {props.user.firstName[0].toUpperCase() +
-          props.user.firstName.slice(1, props.user.firstName.length) +
-          ' ' +
-          props.user.lastName[0].toUpperCase() +
-          props.user.lastName.slice(1, props.user.lastName.length)}
+        <div className='name'>{formatFirstAndLastName(props.user)}</div>
```

Comment:
Awesome! Can you also replace the [code here](https://github.com/SCE-Development/Core-v4/blob/05287caeb4f09ad7487212ea6f529261364c5a87/src/Pages/Profile/MemberView/Profile.jsx#L25) with your new function call?